test/server/storage/loadPaths.json

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
according to our test structures, this resource file should be in directory test/server/storage/safestorage